### ISSUE TITLE
remove navigation mode hint when in terminal accessibility mode

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#172011]
- Since we do not use ctrl+up and down arrow anymore, the hint to use ctrl+up and down arrow to navigate should be removed.

_Originally posted by @pawin35 in https://github.com/microsoft/vscode/issues/169853#issuecomment-1399384962_

**Comment by meganrogge** [comment]
related #170988

**Comment by meganrogge** [comment]
to verify, focus a terminal with a screen reader on. make sure you don't hear a hint about using the up and down arrow keys
